HW1/P6/P6.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import division
+import numpy as np
+import pyspark
+
+sc = pyspark.SparkContext(appName = "Spark1")
+
+allWords = sc.textFile('shakespeare.txt') # I copied the text file to my local directory and deleted the beginning and ending legal non-Shakespeare business
+
+allWords = allWords.map(lambda x: np.array(x.split(' '))).flatMap(lambda x: x[x != '']) # split text by line then remove unnecessary white spaces
+
+allWords = allWords.filter(lambda x: x.isupper() == False) # Filter for all caps and all numbers
+allWords = allWords.filter(lambda x: x.isdigit() == False)
+
```

Comment:
I think you're supposed to also remove words which are all uppercase AND ending with a period.